website/snippets/_mesh-cycle-detection.md

```diff
@@ -1,6 +1,4 @@
-Currently, the default behavior for "project" dependencies enforces that these relationships only go in one direction, meaning that the `jaffle_finance` project could not add a new model that depends, on any public models produced by the `jaffle_marketing` project. dbt will check for cycles across projects and raise errors if any are detected.
-
-However, many teams may want to be able to share data assets back and forth between teams. _We've added support for enabling bidirectional dependencies across projects, currently in beta_. 
+You can enable bidirectional dependencies across projects so these relationships can go in either direction, meaning that the `jaffle_finance` project can add a new model that depends on any public models produced by the `jaffle_marketing` project.
```

Comment:
Add in: "so long as the new dependency does not introduce any node-level cycles." Meaning: dbt will still validate that the DAG is a DAG, at the node-level, just not a _DAG of projects_.